dcat/index.html

```diff
@@ -2128,10 +2124,10 @@ <h4>Property: endpoint description</h4>
             <table class="definition">
                 <thead><tr><th>RDF Property:</th><th><a href="http://www.w3.org/ns/dcat#endpointDescription">dcat:endpointDescription</a></th></tr></thead>
                 <tbody>
-                <tr><td class="prop">Definition:</td><td>A description of the service end-point, including its operations, parameters etc.</td></tr>
+                <tr><td class="prop">Definition:</td><td>A description of the services available via the root endpoint and its children, including their operations, parameters etc.</td></tr>
```

Comment:
Hmm. 'root endpoint and its children' introduces some new concepts, and assumptions about the structure of data service endpoints. I don't think we want to go there.

Reply:
I think we want to talk about these things with terminology that is accurate. Most web services offer multiple endpoints, one of which is the root endpoint.